### ISSUE TITLE
Higher-Level Review | Refine Sentry error monitoring dashboards

### ISSUE THREAD

**Issue by saderagsdale** [department-of-veterans-affairs/va.gov-team#58172]
## Value Statement
**_As a_** software developer
**_I want to_** refine the Sentry dashboard for the Higher-Level Review form
**_So that_** I can easily easy spot errors that need attention and remove noise from the dashboard.

---
## Background Context
We have a Sentry dashboard for our benefits and decision reviews portfolio here: http://sentry.vfs.va.gov/organizations/vsp/dashboard/12/?environment=production. It is still a working draft. The queries for each widget need to be refined so that they capture all errors stemming from their respective forms, and exclude errors that aren't actually errors or that don't actually require follow up. Ultimately, we want this Sentry dashboard to show all true errors stemming from each form and feature in our benefits and decision reviews portfolio. This will help us understand the health of our portfolio and help us know how to allocate engineering resources toward maintenance.

N.B. This ticket is **not** about fixing the errors surfaced by the widgets for HLR. This ticket is about refining the widgets for HLR so that they show all true errors stemming from HLR. Once this ticket is complete, we will know what is wrong with HLR and can follow up accordingly.

## Acceptance Criteria
- [ ] Dashboard has refined widgets.
- [ ] All relevant stakeholders are made aware of these widgets. 

## Out of scope
- Bug fixes from errors identified as the widgets are refined. Once this ticket is complete, we will know what is wrong and can follow up accordingly.
- Notifying the platform team of systemic errors

## Tasks
- [ ] Go through front end and back end code and ensure that all code paths are covered by the widget's query. 
- [ ] Go through the list of errors produced by this query (looking at the past 90 days), and exclude errors that aren't errors or that don't actually require follow up.
- [ ] Make sure all widgets are using this same query.

---
## Definition of Ready 
- [ ] Know what to include and exclude from the dashboard
- [ ] Have an agreement on what visual representation works best for developers

## Definition of Done
- [ ] The final result is reviewed and approved by FE and BE developers
- [ ] The dashboard has been created and is successfully displaying relevant data in an easy-to-read and actionable format
- [ ] Make revisions to the [Logging and Monitoring Framework](https://docs.google.com/document/d/1TqBQmbYGF7Xvjzs_2q83qDuoKd9022n153wVl8V4MmI/edit#heading=h.wb43vjfgcjqq) as needed and report back to Jared/Kevin.
- [ ] The user story has been reviewed and approved by all relevant stakeholders, including product and development.

**Comment by Mottie** [comment]
@saderagsdale I think this work is mostly done. I updated the [Higher-Level Review dashboard](https://vagov.ddog-gov.com/dashboard/kiy-k3e-5ta/benefits-higher-level-reviews?view=spans) around the same time as I updated NOD's dashboard in https://github.com/department-of-veterans-affairs/va.gov-team/issues/63147. So we can close this ticket.
